CHANGELOG.md

```diff
@@ -9,4 +9,4 @@ Changes should start with one of the following tags:
 
 ## v0.14 (unreleased)
 
-- ...
+- [changed] Removed `enum` in `sensors.account_balance`
```

Comment:
Maybe reword this to:

    [changed] The value in `sensors.account_balance` can now be any ISO 4217 string